packages/kanso-app-settings/build/compile.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
 function get_doc(ddoc_url, settings, callback) {
     couchr.get(ddoc_url, function(err, resp){
         if (err && err.error === 'unauthorized') return auth(ddoc_url, settings, callback);
-        if (err && err.error === 'not_found') return callback(null, {}); // no design doc. First push.
+        if (err && err.response.statusCode === 404) return callback(null, {}); // no design doc. First push.
```

Comment:
Filed at https://github.com/kanso/kanso-app-settings/issues/3, but I'm not convinced my idea about the `Connection: close` header being set is entirely accurate.